config/manager/kustomization.yaml

```diff
@@ -12,5 +12,5 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 images:
 - name: controller
-  newName: controller
-  newTag: latest
```

Comment:
I think we .gitignored this but perhaps we neglected to remove it from the cache. It still seems to be getting included.

Reply:
Will sort it out, good catch!